hw/ip/lc_ctrl/doc/_index.md

```diff
@@ -27,6 +256,234 @@ Parameter                   | Default (Max)         | Top Earlgrey | Description
+If two requests arrive simultaneously, the TAP interface is given priority.
+
+The request interface consists of 4 registers (**TODO: add CSR refs**):
+
+1. Target: Specifies the target state to which the agent wants to transition
+2. Token: Any necessary token for the transition.
+3. Start: Start the life cycle transition
+4. Status: Indicates whether the requested transition succeeded.
+If the transition fails, report the cause.
+
+See diagram below
+
+![LC Request Interface](lc_ctrl_request_interface.svg)
+
+#### Hardware Mutex
+
+In order to claim the hardware mutex, a non-zero value must be written to the claim register (**TODO: add CSR ref**).
+If the value written is read back, then the mutex is claimed, and the interface that won arbitration can continue operations.
+If the value is not read back, then the requesting interface should wait and try again later.
+
+When an agent is done with the mutex, it releases the mutex by explicitly writing a 0 to the claim register.
+This resets the mux to select no one and also holds the request interface in reset.
+
+#### Escalation Handler
+
+The life cycle controller also handles two escalation requests from the alert manager.
+When the first escalation request channel is asserted, the life cycle controller permanently asserts the ESCALATE_EN life cycle signal.
+When the second escalation request channel is asserted, the life cycle controller moves the life cycle state into a "virtual" SCRAP state.
+This transition is not permanent, and will clear upon the next power cycle.
+
+### OTP Collateral
+
+The following is a list of all life cycle related collateral stored in OTP.
+Most collateral also contain associated metadata to indicate when the collateral is restricted from further software access, see [accessibility summary]({{< relref "#otp-accessibility-summary-and-impact-of-provision_en" >}}) for more details.
+Since not all collateral is consumed by the life cycle controller, the consuming agent is also shown.
+
+{{< snippet "hw/ip/lc_ctrl/doc/lc_ctrl_otp_collateral.md" >}}
+
+The TOKENs and KEYS are considered secret data and are stored in [wrapped format]({{< relref "#conditional-transitions">}}).
+Before use, the secrets are unwrapped.
+
+The SECRET0_DIGEST and SECRET2_DIGEST are the digest values computed over the secret partitions in OTP holding the tokens and root keys.
+As described in more detail in the [OTP controller specification]({{< relref "hw/ip/otp_ctrl/doc/_index.md#direct-access-memory-map">}}), these digests have a non-zero value once the partition has been provisioned and write/read access has been locked.
+
+#### ID State of the Device
+
+If the SECRET2_DIGEST is zero, the device is considered to have "blank" ID state, in which case the CREATOR_ROOT_KEY_* (in OTP) and CREATOR_DIV_KEY (in FLASH) can be written by software.
+All consumers of these keys are supplied with an invalid all-zero value unless the .
```

Comment:
unless the? is that WIP?

Reply:
uh these are trailing words I forgot to trim away. thanks for noting that.